standalone/node/src/chain_spec.rs

```diff
@@ -693,31 +603,64 @@ fn testnet_genesis(
 		},
 		dkg_proposals: DKGProposalsConfig { initial_chain_ids, initial_r_ids, initial_proposers },
 		bridge_registry: Default::default(),
-		asset_registry: AssetRegistryConfig {
-			asset_names: vec![],
-			native_asset_name: b"tTNT".to_vec().try_into().unwrap(),
-			native_existential_deposit: tangle_runtime::EXISTENTIAL_DEPOSIT,
-		},
-		hasher_bn_254: HasherBn254Config {
-			parameters: Some(bn254_x5_3_params.to_bytes().try_into().unwrap()),
-			phantom: Default::default(),
-		},
-		merkle_tree_bn_254: MerkleTreeBn254Config {
-			phantom: Default::default(),
-			default_hashes: None,
-		},
-		v_anchor_verifier: v_anchor_verifier_config,
-		v_anchor_bn_254: VAnchorBn254Config {
-			max_deposit_amount: 1_000_000 * UNIT,
-			min_withdraw_amount: 0,
-			vanchors: vec![(0, 2)],
-			phantom: Default::default(),
-		},
 		im_online: ImOnlineConfig { keys: vec![] },
 		eth_2_client: Eth2ClientConfig {
 			// Vec<(TypedChainId, [u8; 32], ForkVersion, u64)>
 			networks: vec![eth2_mainnet_genesis_config, eth2_goerli_genesis_config],
 			phantom: PhantomData,
 		},
+		nomination_pools: Default::default(),
+		transaction_payment: Default::default(),
+		// EVM compatibility
+		evm_chain_id: EVMChainIdConfig { chain_id },
+		evm: EVMConfig {
+			accounts: {
+				let mut map = BTreeMap::new();
+				map.insert(
+					// H160 address of Alice dev account
+					// Derived from SS58 (42 prefix) address
+					// SS58: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+					// hex: 0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d
+					// Using the full hex key, truncating to the first 20 bytes (the first 40 hex
+					// chars)
+					H160::from_str("d43593c715fdd31c61141abd04a99fd6822c8558")
+						.expect("internal H160 is valid; qed"),
+					fp_evm::GenesisAccount {
+						balance: U256::from_str("0xffffffffffffffffffffffffffffffff")
+							.expect("internal U256 is valid; qed"),
+						code: Default::default(),
+						nonce: Default::default(),
+						storage: Default::default(),
+					},
+				);
+				map.insert(
+					// H160 address of CI test runner account
+					H160::from_str("6be02d1d3665660d22ff9624b7be0551ee1ac91b")
+						.expect("internal H160 is valid; qed"),
+					fp_evm::GenesisAccount {
+						balance: U256::from_str("0xffffffffffffffffffffffffffffffff")
+							.expect("internal U256 is valid; qed"),
+						code: Default::default(),
+						nonce: Default::default(),
+						storage: Default::default(),
+					},
+				);
```

Comment:
Do we need this? we do not have a CI Test runner, lets remove it.

Reply:
Yep